.travis.yml

```diff
@@ -0,0 +1,30 @@
+sudo: required
+
+addons:
+  apt:
+    sources:
+    - ubuntu-toolchain-r-test
+    packages:
+    - python3-pip
+    - python3.5
+    - g++-7
+
+language: c
+
+before_install:
+  - export CC=gcc-7
+  - export CXX=g++-7
+  - sudo ln -sf /usr/bin/python3.5 /usr/bin/python3
+  - sudo pip3 install meson
+  - wget https://github.com/ninja-build/ninja/releases/download/v1.8.2/ninja-linux.zip
+  - unzip ninja-linux.zip
+  - sudo cp ninja /usr/bin
+  - wget http://http.debian.net/debian/pool/main/c/cyrus-sasl2/cyrus-sasl2_2.1.27~101-g0780600+dfsg.orig.tar.xz
+  - tar xfv cyrus-sasl2_2.1.27~101-g0780600+dfsg.orig.tar.xz
+  - pushd cyrus-sasl2-2.1.27~101-g0780600+dfsg.orig && ./autogen.sh && make && sudo make install && popd
+
+install:
+  - sudo apt install -y libssl-dev libsqlite3-dev libbz2-dev zlib1g-dev
+
+script:
+  - mkdir -p builddir && cd builddir && meson && ninja -v
```

Comment:
This would ideally be `make build` and later on when we have tests implemented `make build && make test`.